django/applications/catmaid/control/sampler.py

```diff
@@ -1286,7 +1288,7 @@ def link_added_node(data, new_nodes):
 
 @api_view(['GET'])
 @requires_user_role([UserRole.Browse])
-def list_domain_intervals(request, project_id, domain_id):
+def list_domain_intervals(request, project_id, domain_id): # FIXME: list_domain_intervals is defined twice
```

Comment:
Let's keep this implementation and remove the other one. This one here seems more complete.